kolibri/plugins/learn/assets/src/state/mutations/index.js

```diff
@@ -32,12 +32,22 @@ export default {
   LEARN_SET_MEMBERSHIPS(state, memberships) {
     state.learnAppState.memberships = memberships;
   },
-  SET_TOPIC_PROGRESS(state, progressArray) {
+  SET_NODE_PROGRESS(state, progressArray) {
```

Comment:
super minor note - `SET_NODE_PROGRESS` and `SET_RECOMMENDED_NODES_PROGRESS` have inconsistent pluralization but the same parameter signature